src/consensus/aft/raft.h

```diff
@@ -799,6 +795,8 @@ namespace aft
           }
           committable_indices.push_back(index);
           start_ticking_if_necessary();
+
+          should_sign = false;
```

Comment:
Why is should_sign being set here?

Reply:
Because I tried to revert those changes by hand, and missed this. Removed it now.